build.sh

```diff
@@ -14,6 +14,7 @@ base=$(pwd)
 GOCMD=${GOCMD:-"go"}
 PLUGIN_DIR=${PLUGIN_DIR:-"/tmp/plugins/"}
 CONTAINER_BUILD=${CONTAINER_BUILD:-false}
+BUILD_ARGS=${BUILD_ARGS:-'-buildvcs=false'}
```

Comment:
I think it might be better to default this to '' and set `-buildvcs=false` only during CI for now

Reply:
That'll be easy, I'll do that first thing on Monday. Another thing that I thought about was to try `rm -rf .git` before building, which might work based on what I read. But that way I think you also lose the VCS info in the binaries and I think your suggestion will be the most elegant anyway.

Reply:
When I first read this I was concerned this approach could break the CI system via GHA, but I was thinking of this bit from sg-bridge: https://github.com/infrawatch/sg-bridge/blob/master/.github/workflows/linter.yml#L10-L29

If a `rm -rf .git/` is required, then it'd be good to make sure that job setup is done in a way where it would wouldn't make sense to have to interact with git after it was done (like in the above linting auto-fix scenario with sg-bridge).

I lack concern now :)